zcash_client_sqlite/src/wallet/init.rs

```diff
@@ -411,6 +411,87 @@ impl<P: consensus::Parameters> RusqliteMigration for WalletMigration2<P> {
     }
 }
 
+struct WalletMigrationAddTxViews;
+
+migration!(
+    WalletMigrationAddTxViews,
+    "282fad2e-8372-4ca0-8bed-71821320909f",
+    ["be57ef3b-388e-42ea-97e2-678dafcf9754"],
+    "Add views over transaction & note data."
+);
+
+impl RusqliteMigration for WalletMigrationAddTxViews {
+    type Error = WalletMigrationError;
+
+    fn up(&self, transaction: &Transaction) -> Result<(), WalletMigrationError> {
+        transaction.execute_batch(
+            "CREATE VIEW IF NOT EXISTS v_tx_sent_notes AS
+            SELECT transactions.id_tx         AS id,
```

Comment:
We should be consistent in how these are referenced:
```suggestion
            SELECT transactions.id_tx         AS id_tx,
```